addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/ReplacerParam.java

```diff
@@ -306,19 +326,41 @@ protected void updateConfigsImpl(int fileVersion) {
             case NO_CONFIG_VERSION:
                 // Handle unversioned to versioned update
                 parseReplacerRules();
-                if (getRule(REPORT_TO_DESC) == null) {
-                    addRule(
-                            new ReplacerParamRule(
-                                    ReplacerParam.REPORT_TO_DESC,
-                                    ReplacerParamRule.MatchType.RESP_HEADER_STR,
-                                    ReplacerParam.REPORT_TO_REGEX,
-                                    true,
-                                    ReplacerParam.REPORT_TO_REPLACEMENT,
-                                    null,
-                                    false));
-                }
+                addIfAbsent(
```

Comment:
Could get the rules from the default list instead of creating new ones.

Reply:
Will do.